RELEASE/scripts/autoscend/auto_deprecation.ash

```diff
@@ -394,6 +394,10 @@ boolean settingFixer()
 	{
 		remove_property("auto_phatloot");
 	}
+	if (property_exists("auto_forcePhatLootToken"))
+	{
+		remove_property("auto_forcePhatLootToken");
+	}
```

Comment:
I wouldn't have bothered with this but it won't hurt I suppose.

Reply:
yea. there was only about 10 minute window in which it could have been created too.
I didn't think i'd get to merge this today.